eips/eip-1588.md

```diff
@@ -19,7 +21,7 @@ This meta-EIP specifies the changes included in the alternative Ethereum hardfor
 - Activation:
   - `Block >= 7280000` on the Ethereum mainnet
 - Included EIPs:
-  - [EIP 1057](./eip-1057.md): ProgPoW, a Programmatic Proof-of-Work
+  - [EIP 1057](@/EIPS/eip-1057.md): ProgPoW, a Programmatic Proof-of-Work
```

Comment:
What kind of syntax is this? is it markdown compatible? Should we consider this for Jekyll? (See #2947)

Reply:
I believe this is a Zola specific thing. The `@/*` style denotes an "internal" link, which as I understand it is mapped to an internal post / asset. I believe that style can be manipulated by Zola to achieve certain permalink schemes. The `./*` style denotes a "relative" link and is treated as a simple relative link.

Reply:
The CHANGELOG on Zola for more info: https://github.com/getzola/zola/blob/2d1c95432290478547316522d1f9219ac38e5b13/CHANGELOG.md#breaking-4

Reply:
As with my other comments above, I'm moderately against any change that breaks formatting when viewed in GitHub.